bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/handler/RotelHandler.java

```diff
@@ -1377,6 +1534,53 @@ public void onNewMessageEvent(EventObject event) {
                 case RotelConnector.KEY_UPDATE_MODE:
                 case RotelConnector.KEY_DISPLAY_UPDATE:
                     break;
+                case RotelConnector.KEY_TCBYPASS:
+                    if (RotelConnector.MSG_VALUE_ON.equalsIgnoreCase(value)) {
+                        tcbypass = true;
+                        updateChannelState(CHANNEL_TCBYPASS);
+                    } else if (RotelConnector.MSG_VALUE_OFF.equalsIgnoreCase(value)) {
+                        tcbypass = false;
+                        updateChannelState(CHANNEL_TCBYPASS);
+                    } else {
+                        throw new RotelException("Invalid value");
+                    }
+                    break;
+                case RotelConnector.KEY_BALANCE:
+                    if (RotelConnector.MSG_VALUE_MIN.equalsIgnoreCase(value)) {
+                        balance = minBalanceLevel;
+                    } else if (RotelConnector.MSG_VALUE_MAX.equalsIgnoreCase(value)) {
+                        balance = maxBalanceLevel;
+                    } else {
+                        balance = Integer.parseInt(value);
```

Comment:
I just forgot the most important.
Normally, this code should not work as the specification is mentioning retuned values like L01-15, R01-15, 000
Your code is not considering initial L or R.

Reply:
Fix included in my PR